Dockerfile

```
@@ -53,5 +53,8 @@ RUN set -eux; \
 RUN set -eux; \
 	eval "GOARCH=s390x go build $BUILD_FLAGS -o /go/bin/gosu-s390x"; \
 	file /go/bin/gosu-s390x
+RUN set -eux; \
+	eval "GOARCH=mips64le go build $BUILD_FLAGS -o /go/bin/gosu-mips64le"; \
+	file /go/bin/gosu-mips64le
```

These files are all named according to `dpkg` architecture name conventions (`gosu-$(dpkg --print-architecture)`) to make them easier to download in a cross-platform way:

```suggestion
	eval "GOARCH=mips64le go build $BUILD_FLAGS -o /go/bin/gosu-mips64el"; \
	file /go/bin/gosu-mips64el
```

Also, I think it would be good to move this up between `arm64` and `ppc64`.

---

updated